pkg/volume/gcepd/attacher_test.go

```diff
@@ -310,10 +484,8 @@ type detachCall struct {
 }
 
 type diskIsAttachedCall struct {
-	diskName   string
-	nodeName   types.NodeName
-	isAttached bool
-	ret        error
+	attachedDisks map[types.NodeName][]string
+	ret           error
```

Comment:
better name for this `retErr`? `ret` makes me think of "return value"

Reply:
ack